css/style.css

```diff
@@ -0,0 +1,706 @@
+body {
+  margin: 0;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+/* HEADER */
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5px 2%;
+  border-bottom: 1px solid #e2e2e2;
+  position: fixed;
+  right: 0;
+  left: 0;
+  top: 0;
+  background-color: white;
+  z-index: 1;
+}
+
+.icons a {
+  text-decoration: none;
+  color: black;
+}
+
+.fa-bars {
+  margin-right: 20px;
+}
+
+.nav-button {
+  background-color: #567b95;
+  color: #f6f8fa;
+  font-weight: 600;
+  border: none;
+  padding: 10px;
+  border-radius: 5px;
+  font-size: 10px;
+  cursor: pointer;
+}
+
+/* CONTENT */
+
+.ad {
+  height: 15vh;
+  text-align: center;
+  background-color: #f8f8f8;
+  padding: 1px;
+  color: #ccc;
+  margin-top: 45px;
+}
+
+.hero h4 {
+  text-align: left;
+  font-size: 2.2rem;
+  font-family: 'Arbutus Slab', serif;
+  font-style: italic;
+  width: 600px;
+  margin: 0 auto;
+}
+
+.figure-hero {
+  max-width: 65%;
+  margin: 0 auto;
+}
+
+.hero-img {
+  max-width: 100%;
+  margin: 20px auto 0 auto;
+}
+
+.figure-hero-caption {
+  max-width: 86%;
+  color: #666;
+}
+
+iframe {
+  border: none;
+}
+
+.video-legend {
+  max-width: 86%;
+  color: #666;
+  font-size: 0.9rem;
+  line-height: 20px;
+}
+
+.image-credit {
+  font-size: 13px;
+  color: #888;
+}
+
+.credit-video-legend {
+  font-size: 13px;
+  color: #888;
+}
+
+.author {
+  width: 31%;
+  margin: 40px auto 0 auto;
+}
+
+.date {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.social-media a {
+  background-color: #eee;
+  border: 1px #eee solid;
+  padding: 5px;
+  text-align: center;
+  border-radius: 50%;
+}
+
+.fix a {
+  height: 10px;
+  width: 23px;
+  font-size: 15px;
+}
+
+.social-media a i {
+  width: 18px;
+  color: black;
+}
+
+.author-name {
+  color: black;
+  text-decoration-color: #ccc;
+}
+
+.author h5 {
+  margin: 0;
+}
+
+.date p {
+  font-size: 0.8rem;
+  font-weight: 500;
+}
+
+.articles {
+  width: 44%;
+  margin: auto;
+  font-size: 1.25rem;
+  line-height: 1.875rem;
+  padding: 0 20px;
+}
+
+.subscribe-nyt {
+  text-align: center;
+}
+
+.hr-width {
+  width: 75px;
+}
+
+.subscribe-nyt h6 {
+  margin: 0;
+  margin-bottom: -10px;
+  color: #333;
+  font-size: 10px;
+}
+
+.subscribe-nyt a {
+  text-decoration: none;
+  color: #326891;
+  font-size: 10px;
+  font-weight: 600;
+}
+
+.editor-picks {
+  width: 13%;
+
+  /* border: 1px solid red; */
+  position: absolute;
+  right: 12%;
+  top: 1945px;
+}
+
+.editor-picks h5 {
+  margin: 0;
+  font-size: 11px;
+}
+
+.editor-picks hr {
+  margin: 0;
+  color: #e2e2e2;
+}
+
+.picks {
+  display: flex;
+
+  /* border: 1px solid green; */
+  justify-content: space-around;
+  align-items: center;
+}
```

Comment:
## Inline comments

Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

- [ ]  Check all your inline comments in the entire page and remove them